src/main/kotlin/view/FilterPanel.kt

```diff
@@ -85,7 +104,7 @@ class FilterPanel : View() {
                         min = 0.0
                         max = 100.0
                     }
-                    slider.value = 100.0
+                    slider.value = 50.0
                     slider.valueChangingProperty()
                         .addListener(ChangeListener { _, _, _ -> op(slider.value / 100) })
```

Comment:
Changing this from `slider.value / 100` to `slider.value / 50` makes more sense than multiplying `factor * 2` for me

Reply:
I think the issue is the slider mode should be able to switch from RGB mode to HSV mode, and applies different min/max scales for each mode.